features/backup_create.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+PARSED=$(getopt --options v --longoptions datadir:,dbname:,walmethod: --name "$0" -- "$@" 2> /dev/null)
```

Comment:
This syntax is for GNU getopt, BSD getopt is quite limited compared to the GNU one: https://www.freebsd.org/cgi/man.cgi?getopt(3) (looks like it only supports short one-letter options).